extras/docker/centos/Dockerfile.testing

```diff
@@ -0,0 +1,21 @@
+FROM registry.centos.org/centos/centos:latest
+MAINTAINER Heketi Developers <heketi-devel@gluster.org>
+
+LABEL version="0.8"
+LABEL description="Heketi based on the packages from the CentOS Storage SIG in TESTING"
+
+# install dependencies, build and cleanup
+RUN yum -y install centos-release-gluster && \
+    yum -y --enablerepo=centos-gluster*-test install heketi heketi-client && \
```

Comment:
Can we avoid duplicate dockerfiles by using a variable here?

Reply:
I do not think the CentOS Container Pipeline provides that capability.

Reply:
Oh. That's unfortunate. I guess there's no way to pass environment variables via that job spec below.